dango/app/src/proposal_preparer.rs

```diff
@@ -68,24 +81,28 @@ impl ProposalPreparer {
         let _handle = thread::spawn(move || {
             let update_func = || -> Result<(), ProposerError> {
                 // Copy the params to unlock the mutex.
-                let params = thread_params.read_access().clone();
+                let mut params = thread_params.read_access().clone();
                 if params.is_empty() {
                     return Ok(());
                 }
 
+                // Set the encoding to base64 to match the oracle contract.
+                params.push(("encoding", "base64".to_string()));
+                // Set the parsed to false since we don't use parsed data.
+                params.push(("parsed", "false".to_string()));
+
                 // Retrieve vaas from pyth node.
                 let vaas = reqwest::blocking::Client::builder()
                     .timeout(REQUEST_TIMEOUT)
                     .build()?
-                    .get(format!("{PYTH_URL}/api/latest_vaas"))
+                    .get(format!("{PYTH_URL}/v2/updates/price/latest"))
                     .query(&params)
                     .send()?
-                    .json::<Vec<Binary>>()?;
+                    .json::<LatestVaaResponse>()?
+                    .binary
+                    .data;
 
-                tracing::info!(
-                    "Prepare proposal: fetched latest vaas - len: {}",
-                    vaas.len()
-                );
+                info!(len = vaas.len(), "Prepare proposal: fetched latest Vaas",);
```

Comment:
Remove the trailing comma. Also, "VAAs" instead of "vaas":

```suggestion
                info!(len = vaas.len(), "Prepare proposal: fetched latest VAAs");
```